packages/amplify-environment-parameters/API.md

```diff
@@ -19,6 +19,11 @@ export type IEnvironmentParameterManager = {
     hasResourceParamManager: (category: string, resource: string) => boolean;
     getResourceParamManager: (category: string, resource: string) => ResourceParameterManager;
     save: (serviceUploadHandler?: ServiceUploadHandler) => Promise<void>;
+    getMissingParameters: () => Promise<{
```

Comment:
If you export `ResourceParameter`, this file will show a named type as the resolution value of the promise instead of an inline object.

Reply:
I didn't want to commit to a name for the type (insert hardest problems in cs joke). Given that it's only 3 values, I figured it was better to avoid polluting the namespace with something generic like `ResourceParameter`.

Reply:
I think that this should be a named type.